src/App.js

```diff
@@ -8,23 +10,25 @@ import NoMatch from './components/NoMatch';
 import './App.css';
 
 const App = () => (
-  <Router basename={process.env.PUBLIC_URL}>
-    <NavBar />
-    <Switch>
-      <Route exact path="/">
-        <RocketsPage />
-      </Route>
-      <Route path="/missions">
-        <MissionsPage />
-      </Route>
-      <Route path="/profile">
-        <ProfilePage />
-      </Route>
-      <Route path="*">
-        <NoMatch />
-      </Route>
-    </Switch>
-  </Router>
+  <Provider store={store}>
```

Comment:
The `<Provider>` component makes the Redux store available to any nested components that need to access the Redux store. Since our components  can be connected to the store, instead of wrapping all components in the provider, let's try to render the `<Provider>` at the top level, with the entire app's component tree inside of it. You can do this in  the index.js file like this 
```suggestion
 ReactDOM.render(
  <React.StrictMode>
    <Provider store={store}>
      <App />
    </Provider>
  </React.StrictMode>,
  document.getElementById('root'),
);
```